recipes/openpiv/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.20.2" %}
+
+package:
+  name: openpiv
+  version: {{ version }}
+
+source:
+  fn: openpiv-{{ version }}.tar.gz
+  url: https://github.com/OpenPIV/openpiv-python/archive/{{ version }}.tar.gz
+  sha256: 39495fdd7f67cf88469b7c1684596ca0f823dd06e2efa28f1daa08c0ee2e364b
+  
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Make sure that we are using that we have merged into the latest commit on `master`. It may be that your fork is very out-of-date.